### PR TITLE
Update README file

### LINKED ISSUE #8550: Clean up the README file
### Intent

Current README file is a bit of a mess right now. 
Just to name a few things:
- No "Getting started" section that would allow to start using it right of the bat (like on website)
- Link to documentation, arguably one of the most important things, is buried deep in `Helpful Links`.
- Links to (now closed) community forums and unused build server are still there

### Motivation

README file is the first thing people see when visiting a repo, so fixing the issues listed above and adding new information (and a few links to relevant resources) will help new and existing users with using MonoGame.

I'm working on this right now, you can see the progress [in my fork](https://github.com/Maniekko/MonoGame/blob/develop/README.md). 
Let me know if i should open a draft PR, since current version  in my fork **is far from being completed.**

### PR DESCRIPTION
**Pull request by Maniekko**
Fixes #8550

This PR aims to clean up and update info in README.md file. You can always view how the WIP looks like right now [here](https://github.com/Maniekko/MonoGame/blob/develop/README.md) 
- Added a logo (from [MonoGame.Logo](https://github.com/MonoGame/MonoGame.Logo) repo) and moved the badges to the top
- Removed outdated links to community forums, replacing them with GitHub discussions page
- Removed the Build Status chapter, the Build badge is usually enough to show that Actions are used.

This PR is marked as a draft since there's some things to do:
\- Add a proper "Getting started" chapter, instead of dry links to docs
\- Make sure all information is up to date
\- Maybe prettify existing info?

:warning: My knowledge of English is pretty limited, if you see a typo (or just want to help) feel free to open a PR **in my fork**.

### CODE REVIEW

**Comment by tomspilman** [comment]
Maybe link this in the platforms for console access.

https://docs.monogame.net/articles/console_access.html

**Comment by SimonDarksideJ** [comment]
Made a few "Quality of Life" improvements as I'm really liking this redesign, namely:

* Added Toms suggestion for linking to the consoles doc
* Fixed links in the Getting Started section (and added a ref to the new hot to content
* Added Patreon to the "Helpful links" section
* Added a Samples section, as this was missing, shockingly.

**Review by SimonDarksideJ — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -32,17 +32,42 @@ We support a growing list of platforms across the desktop, mobile, and console s
...
   * Android 6.0 and up (OpenGL)
   * iPhone/iPad 10.0 and up (OpenGL)
-* Consoles (for registered developers)
+* [Consoles (for registered developers)](https://docs.monogame.net/articles/console_access.html)
   * PlayStation 4
   * PlayStation 5
   * Xbox One (XDK only) (GDK coming soon)
   * Nintendo Switch
 
+## Getting started
```
Comment (tomspilman) [inline README.md:41]: Maybe change this title to something else?  Or the first "Getting started" below to something else.  Feels weird having two "Getting started"s right on each other.

Maybe `##Documentation` instead?

File: README.md [hunk 2]
```diff
@@ -32,17 +32,42 @@ We support a growing list of platforms across the desktop, mobile, and console s
...
+- [Getting started →](https://docs.monogame.net/articles/getting_started/index.html)
+- ["How To" Guides →](https://docs.monogame.net/articles/getting_to_know/howto/)
+- [Documentation Hub →](https://docs.monogame.net/)
+- [API Reference →](https://docs.monogame.net/api/index.html)
+- [Community Tutorials →](https://docs.monogame.net/articles/tutorials.html)
+
+## [Samples](https://github.com/MonoGame/MonoGame.Samples)
```
Comment (tomspilman) [inline README.md:48]: This is already linked in the paragraph below... remove this link.

File: README.md [hunk 3]
```diff
@@ -32,17 +32,42 @@ We support a growing list of platforms across the desktop, mobile, and console s
...
+
+Check out the awesome ["full game" samples](https://github.com/MonoGame/MonoGame.Samples) maintained by the MonoGame team:
```
Comment (tomspilman) [inline README.md:50]: How about just 'game samples` as the link without the quotes.

File: README.md [hunk 4]
```diff
@@ -32,17 +32,42 @@ We support a growing list of platforms across the desktop, mobile, and console s
...
+|-|-|
+Supported on all platforms | Supported on all platforms |
+[![Platformer 2D Sample](https://raw.githubusercontent.com/MonoGame/MonoGame.Samples/refs/heads/3.8.2/Images/Platformer2D-Sample.png)](https://github.com/MonoGame/MonoGame.Samples/blob/3.8.2/Platformer2D/README.md) | [![NeonShooter Sample](https://raw.githubusercontent.com/MonoGame/MonoGame.Samples/refs/heads/3.8.2/Images/NeonShooter-Sample.png)](https://github.com/MonoGame/MonoGame.Samples/blob/3.8.2/NeonShooter/README.md) |
+The [Platformer 2D](https://github.com/MonoGame/MonoGame.Samples/blob/3.8.2/Platformer2D/README.md) sample is a basic 2D platformer pulled from the original XNA samples and upgraded for MonoGame.| [Neon Shooter](https://github.com/MonoGame/MonoGame.Samples/blob/3.8.2/NeonShooter/README.md) Is a graphically intensive twin-stick shooter with particle effects and save data from Michael Hoffman |
+|||
+
+| [Auto Pong Sample](https://github.com/MonoGame/MonoGame.Samples/blob/3.8.2/AutoPong/README.md) | [Ship Game 3D](https://github.com/MonoGame/MonoGame.Samples/blob/3.8.2/ShipGame/README.md) |
+|-|-|
+| Supported on all platforms | GL / DX/ iOS / Android |
```
Comment (tomspilman) [inline README.md:61]: How about `Desktop and mobile` instead of jargon like `GL / DX / iOS / Android`?

**Comment by Maniekko** [comment]
I think this is now ready to be merged

File: README.md (hunk 4 above)
Comment (Maniekko) [reply README.md]: Isn't `Desktop and Mobile` already `All platforms`?

**Comment by Maniekko** [comment]
...For some reason it breaks CI?

**Comment by SimonDarksideJ** [comment]
Those errors are not related to this pr, so they are fine

**Review by SimonDarksideJ — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (tomspilman) [reply README.md]: No...  Desktop, mobile, and console would be all platforms.

**Pull request merged**